core/src/replicator.rs

```diff
@@ -102,24 +114,24 @@ impl Replicator {
     /// * `ledger_path` - path to where the ledger will be stored.
     /// Causes panic if none
     /// * `node` - The replicator node
-    /// * `leader_info` - ContactInfo representing the leader
+    /// * `network_entrypoint` - ContactInfo representing an entry into the network
```

Comment:
Can you name that `cluster_entrypoint` to be consistent with the book's terminology?